package.json

```diff
@@ -66,7 +66,7 @@
     "patternfly": "3.20.0",
     "pluralize": "5.0.0",
     "rxjs": "5.4.1",
-    "syndesis.data.mapper": "https://github.com/syndesisio/syndesis-datamapper-ui.git#9476d91ec03c0e4885f3005876c67216130ef98d",
+    "syndesis.data.mapper": "https://github.com/atlasmap/atlasmap-ui.git",
```

Comment:
wait a moment, after setting that i'm receiving build errors:

`
yarn start
yarn start v0.22.0
$ npm run build

> syndesis.data.mapper@1.0.0 build /Users/jason/Documents/code/md/ipaasui/syndesis-ui/node_modules/syndesis.data.mapper
> tsc -p ./

../@types/lodash/index.d.ts(19449,15): error TS2428: All declarations of 'WeakMap' must have identical type parameters.
../typescript/lib/lib.es2015.collection.d.ts(45,11): error TS2428: All declarations of 'WeakMap' must have identical type parameters.
../typescript/lib/lib.es2015.iterable.d.ts(124,11): error TS2428: All declarations of 'WeakMap' must have identical type parameters.
../typescript/lib/lib.es2015.symbol.wellknown.d.ts(133,11): error TS2428: All declarations of 'WeakMap' must have identical type parameters.
`